robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -130,93 +134,155 @@ def init_serial():
     rospy.logerr('Incorrect MCU connected, terminating listener')
     sys.exit(0)
 
+requests = {
+    'PDS T 1' : ['ON'],
+    'PDS T 0' : ['OFF'],
+    #'who' : ['pds'],
+    'PDS M 1 1' : ['toggling'],
+    'PDS M 1 0' : ['toggling'],
+    'PDS M 2 1' : ['toggling'],
+    'PDS M 2 0' : ['toggling'],
+    'PDS M 3 1' : ['toggling'],
+    'PDS M 3 0' : ['toggling'],
+    'PDS M 4 1' : ['toggling'],
+    'PDS M 4 0' : ['toggling'],
+    'PDS M 5 1' : ['toggling'],
+    'PDS M 5 0' : ['toggling'],
+    'PDS M 6 1' : ['toggling'],
+    'PDS M 6 0' : ['toggling']
+}
+def handle_client(req):
+    global ser # specify that it's global so it can be used properly
+    global reqFeedback
+    global reqInWaiting
+    pdsResponse = ArmRequestResponse()
+    timeout = 0.3 # 300ms timeout
```

Comment:
Not a huge issue but it would be nice to rename this `timeout` or the one on line 23, because it causes confusion especially if you forget that `global` is needed in python. I thought this was overwriting the one above lol. Also renaming them can add a bit more clarity on what the timeout on line 23 is for vs this one

Reply:
Will update `timeout` to `reqTimeout` for clarity.